### PR TITLE
feat: add timestampinfo to requests

### PR DESCRIPTION
**Pull request by marcoscaceres**

### CODE REVIEW

**Comment by marcoscaceres** [comment]
rebasing...

File: index.js [hunk 1]
```diff
@@ -18,6 +18,8 @@ app.use(require("errorhandler")(errorhandlerOptions));
 // bibrefs
 app.get('/bibrefs', function (req, res, next) {
     var refs = req.query["refs"];
+    res.setHeader("Expires", new Date(Date.now() + 86400000).toUTCString());
```
Comment (tobie) [inline index.js]: Why 24h? Fwiw, the update script runs every hour.

File: index.js [hunk 2]
```diff
@@ -9,7 +9,7 @@ if (process.env.NODE_ENV == "dev" || process.env.NODE_ENV == "development") {
     errorhandlerOptions.dumpExceptions = true;
     errorhandlerOptions.showStack = true;
 }
-
+app.enable("etag");
```
Comment (tobie) [inline index.js]: Given the app is restarted every time there is a change in the DB, we're not going to get anything out of this unless the etags are stored somewhere. If so, where?

File: index.js (hunk 1 above)
Comment (marcoscaceres) [inline index.js]: 24 hours seems like a fairly sane default per spec, no?

File: index.js (hunk 2 above)
Comment (tobie) [inline index.js]: Oh, wait. They're content hashes, no? So that should work.

Comment (marcoscaceres) [inline index.js]: Yeah, I think they are content hashes: 
https://github.com/expressjs/express/blob/3ffceff3ed356a711fce5fee6ed00a03f43affd8/lib/utils.js#L21

File: index.js (hunk 1 above)
Comment (tobie) [inline index.js]: Yeah, It's rare that we get manual update requests nowadays, so I guess the use-case where someone just fixed the db and wants to see the update right away in their spec is rare enough.

**Comment by tobie** [comment]
Alright, this LGTM. Let's give it a spin. Mind messaging spec-prod to let them know we've shipped this?

**Comment by marcoscaceres** [comment]
> Mind messaging spec-prod to let them know we've shipped this?

Will do! Thanks! 

Will this be deployed straight away? I'm cooking up some new goodness based on this :)

**Comment by marcoscaceres** [comment]
YAY! I can see the 304 in https://specref.herokuapp.com/bibrefs?refs=dom 💃

**Comment by tobie** [comment]
Yup, and the proper cache headers are there too. Awesome!

**Pull request merged**